Client.js

```diff
@@ -11,19 +11,48 @@ function Client(accessToken) {
   });
 
   this.calendarResource = new CalendarResource(this);
+  this.buildingResource = new BuildingResource(this);
 }
 
 function CalendarResource(client) {
   this.client = client;
   this.resourcePath = '/directory/v1/customer/my_customer/resources/calendars'
 }
 
+function BuildingResource(client) {
+  this.client = client
+  this.resourcePath = '/directory/v1/customer/my_customer/resources/buildings'
+}
+
+function list(qs, cb) {
+  if (typeof qs === 'function') {
+    cb = qs
+    qs = {}
+  }
+  return this.client.request({method: 'GET', uri: this.resourcePath, qs: qs}, cb)
+};
+
+function get(resourceId, qs, cb) {
+  if (!resourceId) {
+    throw new Error('[google-apps-admin-sdk] resourceId must be provided when trying to GET a single resource')
+  }
```

Comment:
If `qs` are provided but not resourceId, we wouldn't hit this error

Reply:
True. This was really just for convenience's sake, and since the other functions in this library do not do any input validation, I've decided to remove it entirely.